grafana/operator/kustomization.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- namespace.yaml
+# - namespace.yaml
```

Comment:
Temporarily commenting this out so that I can add the `sigstore-monitoring` namespace in the charts directly